packages/teams-js/src/public/pages.ts

```diff
@@ -750,4 +750,94 @@ export namespace pages {
       return runtime.supports.pages ? (runtime.supports.pages.appButton ? true : false) : false;
     }
   }
+
+  /**
+   * Parameters for the NavigatewithinApp API
+   *
+   * @beta
+   */
+  export interface NavigateWithinAppParams {
+    /**
+     * @deprecated
+     * As of 2.0.0, please use {@link app.PageInfo.id | app.Context.page.id} instead
+     *
+     * Developer-defined ID of the Page to navigate to within the application (Formerly EntityID)
+     */
+    pageId: string;
+
+    /**
+     * @deprecated
+     * As of 2.0.0, please use {@link app.PageInfo.subPageId | app.Context.page.subPageId} instead
+     *
+     * Optional developer-defined ID describing the content to navigate to within the Page. This will be passed
+     * back to the application via the Context object.
+     */
+    subPageId?: string;
+  }
+
+  /**
+   * Provides functions for navigating without your own application ID.
+   *
+   * @beta
+   */
+  export namespace navigate {
```

Comment:
We had a side chat about this, but I think it would make sense to name this subcapability "currentApp", and then name the functions "navigateTo" and "navigateToDefaultPage".

I think this will not overlap with how all of pages is for navigation, and will also make it more clear what the difference is between this subcapability and the parent pages capability.

Reply:
Resolved in latest commit